tests/TESTFPC1.MLC

```diff
@@ -96,12 +98,12 @@ LHT1     DC    LH'-567678867789987898'
          MVC   DHEX,=CL32' '
          LA    R3,L'&VAL
          CALL  CVTTOHEX,(WORK16,DHEX,(R3))
-         WTO   MF=(E,WTOMSG)
+         PUT   RT1OUT,WTOMSG                                             #540
          IF    (CP,PRC,NE,=P'0')
                MVC DRC,=X'40202020'
                ED  DRC,PRC+6
-               WTO MF=(E,WTOPRC)
-               ABEND 102
+               PUT RT1OUT,WTOPRC                                         #540
+               RT1 ABORT               Close file and abend              #540
```

Comment:
This condition in this IF now ends with ABEND 101, not ABEND 102. Same as preceding IF. Just pointing this out. Okay to leave as is. Let me know what you think.

Reply:
I think it matters little which abend code is used. If it happens someone is going to have to dig for the root cause anyway. But I can see the bonus of having a more precise indication. I could change the macro to support a variable abend code number. I'll go ahead and see if I can get that done with limited effort.